wire-ios/Wire-iOS/Wire-Info.plist

```diff
@@ -186,8 +167,6 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>UIViewControllerBasedStatusBarAppearance</key>
-	<true/>
```

Comment:
`true` is the [default value](https://developer.apple.com/documentation/bundleresources/information_property_list/uiviewcontrollerbasedstatusbarappearance), so theses lines can be removed.